web/src/index.css

```diff
@@ -108,6 +108,17 @@ a:focus-visible, button:focus-visible {
   @apply outline-dashed outline-muted-foreground outline-offset-2 rounded-3xl outline-1
 }
 
+@layer components {
+  aside ul li a {
```

Comment:
When updating the aside element to nav, this needs an update too 🤞

Reply:
🫶